test/manual/definitions/apm/php-agent/php-apache-fpm-wordpress-ubuntu18.json

```diff
@@ -0,0 +1,29 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+        "id": "php-a-f-w-u18",
+        "display_name": "PHP Apache FPM Wordpress",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.micro",
+        "ami_name": "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-????????.1",
+        "user_name": "ubuntu"
+    }],
+
+    "services": [
+        {
+          "id": "wordpress",
+          "display_name": "Debian PHP Apache FPM Wordpress",
+          "local_source_path": "/mnt/open-install-library",
```

Comment:
@Fahmy-Mohammed there are a few more files that need to have the local_source_path updated